File: experiment_jp_test/j2pyrawpy/for__0__HARDY_RAMANUJAN_THEOREM.py
#!/usr/bin/env python
""" generated source for module for__0__HARDY_RAMANUJAN_THEOREM """
class X(object):
    """ generated source for class X """
    @classmethod
    def exactPrimeFactorCount(cls, n):
        """ generated source for method exactPrimeFactorCount """
        count = 0
        if n % 2 == 0:
            count += 1
            while n % 2 == 0:
        if n > 2:
            count += 1
        return count

File: experiment_jp_test/j2pyraw/for__0__HARDY_RAMANUJAN_THEOREM.java
class X { static int exactPrimeFactorCount ( int n ) { int count = 0 ; if ( n % 2 == 0 ) { count ++ ; while ( n % 2 == 0 ) n = n / 2 ; } if ( n > 2 ) count ++ ; return count ; }
 }